victoriametrics.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: victoriametrics
+  version: 1.102.1
+  epoch: 0
+  description: VictoriaMetrics is a fast, cost-effective, and scalable monitoring solution and time series database designed for high performance and reliability. It supports both single-server and clustered installations, providing flexibility for various deployment needs, and integrates well with tools like Grafana for data visualization.
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 996b623585d1945a298d95d9e2ea4f055e2dcd2a
+      repository: https://github.com/VictoriaMetrics/VictoriaMetrics
+      tag: v${{package.version}}
+
+subpackages:
+  - name: ${{package.name}}-vmagent
+    description: vmagent is a tiny agent which helps you collect metrics from various sources, relabel and filter the collected metrics and store them in VictoriaMetrics or any other storage systems via Prometheus remote_write protocol or via VictoriaMetrics remote_write protocol.
+    pipeline:
+      - uses: go/build
+        with:
+          ldflags: -extldflags "-static"
+          toolchaintags: netgo,osusergo,nethttpomithttp2,musl
+          packages: github.com/VictoriaMetrics/VictoriaMetrics/app/vmagent
```

Comment:
static linking is either redundant (as binary is already static anyway) or does not achieve what you think it does (statically links glibc, and shared libraries creating dark matter in our containers unaccounted for with sbom).

some of the tags are redundant as already specified by our default pipeline.

Some look sus - what does musl tag do?

Reply:
Ah that's probably a relic of the fact they're own docker image is alpine, i'll try removing. nethttp one seems like it's just a space saver for them. I can remove static linking as I was just following their build: https://github.com/VictoriaMetrics/VictoriaMetrics/blob/22975d28ae4871a4e6ede1e482b373e13523870b/deployment/docker/Makefile#L43 and it worked without it anyways for the fips side.